Makefile.work

```diff
@@ -128,6 +128,7 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            HTTPS_PROXY=$(https_proxy) \
                            SONIC_ENABLE_SYSTEM_TELEMETRY=$(ENABLE_SYSTEM_TELEMETRY) \
                            EXTRA_JESSIE_TARGETS=$(EXTRA_JESSIE_TARGETS) \
+                           MULTI_ASIC=$(MULTI_ASIC) \
```

Comment:
I am not expecting to have this as a build option. 

the image should be able to adapt in runtime to figure out whether the platform is multi-asic or single asic and adapt to it.